### PR TITLE
Update Senior & Wordpress Engineer roles

### PR DESCRIPTION
**Pull request by ryanboswell**
Making some slight changes to our open roles and job descriptions.

### CODE REVIEW

**Comment by ryanboswell** [comment]
@goldenapples care to double check my copy-paste job here?

**Comment by goldenapples** [comment]
Looks good to me!

File: roles/wordpress-engineer.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # WordPress Engineer
 
-Our WordPress Engineers work on one of the most ambitious WordPress-based projects in the world. We’re doing things no one in their right mind would think of doing — and we share our successes (and failures) through open source projects, blog posts, and talks at conferences.
+Our WordPress Engineer role will get to work on one of the most ambitious WordPress-based projects in the world. We’re doing things no one in their right mind would think of doing — and look forward to sharing our successes through open source projects, blog posts, and talks at conferences.
```
Comment (scribu) [inline roles/wordpress-engineer.md]: That doesn't sound right. It's not a "WordPress Engineer role" that will do the work, but simply a "WordPress Engineer".

**Comment by scribu** [comment]
Why do we have both [lead engineer](https://github.com/fusioneng/tech-docs/blob/master/roles/lead-engineer.md) and [senior engineer](https://github.com/fusioneng/tech-docs/blob/master/roles/senior-engineeer.md)?

File: roles/wordpress-engineer.md (hunk 1 above)
Comment (ryanboswell) [inline roles/wordpress-engineer.md]: Ha, that's a good catch, and precisely why copy/paste isn't always a great thing. I'll get that fixed in  a moment.

**Comment by ryanboswell** [comment]
@scribu lead engineers have more explicit mentoring/leadership responsibilities than a senior engineer, not that a senior engineer shouldn't mentor or lead when the opportunity presents itself.

**Pull request merged**